### PR TITLE
Update update-settings.asciidoc

### PR DESCRIPTION
**Pull request by kcm**
add explanation for ?include_defaults parameter

- Have you signed the [contributor license agreement](https://www.elastic.co/contributor-agreement)?
- Have you followed the [contributor guidelines](https://github.com/elastic/elasticsearch/blob/master/CONTRIBUTING.md)?
- If submitting code, have you built your formula locally prior to submission with `gradle check`?
- If submitting code, is your pull request against master? Unless there is a good reason otherwise, we prefer pull requests against master and will backport as needed.
- If submitting code, have you checked that your submission is for an [OS that we support](https://www.elastic.co/support/matrix#show_os)?
- If you are submitting this code for a class then read our [policy](https://github.com/elastic/elasticsearch/blob/master/CONTRIBUTING.md#contributing-as-part-of-a-class) for that.

### CODE REVIEW

File: docs/reference/cluster/update-settings.asciidoc [hunk 1]
```diff
@@ -11,6 +11,14 @@ GET /_cluster/settings
 --------------------------------------------------
 // CONSOLE
 
+This API only returns settings explicitly defined. The default cluster settings can also be returned using the `?include_defaults` parameter:
```
Comment (cbuescher) [inline docs/reference/cluster/update-settings.asciidoc]: nit: maybe rephrase a bit to make clear only the above (parameterless) call returns only the explicitly defined settings. The API is capable of returning the defaults, as you are just about to explain now.

**Review by cbuescher — APPROVED** [review]
@kcm great addition, I left a short maybe a bit nitpicky comment, let me know if you want to change this. I'd also be fine merging as-is I think.

**Review by cbuescher — COMMENTED** [review]
On a second glance: the [get settings API documentation](https://www.elastic.co/guide/en/elasticsearch/reference/master/cluster-get-settings.html) already documents the `include_defaults` parameter with the same console example, maybe that should be sufficient? I'm on the fence if this is needed twice, let me know what you think.

**Comment by kcm** [comment]
@cbuescher agreed on wording, was on site and dashed it off quickly. The team lamented that we don't mention the option, and I think Cluster Settings used to be a single page, so perhaps something was lost in the split. My latest change should clarify that. :)

Let me know.

**Comment by cbuescher** [comment]
@kcm revisiting this again, I left a comment a few days ago. I think I liked the previous version better, even with the repetition. Linking to the get-settings docs it fine, but to me its not clear now why I would need to see "Cluster Get Settings". The brackets suggest this context suggest this info is something of an aside, but I think this interupts the flow of the main page here. I'd probably prefer sth. like "This API call only returns settings explicitly defined, but there are options to also <<cluster-get-settings,include the default settings.>>. Best of both? Wdyt?

**Comment by cbuescher** [comment]
@kcm any chance of moving this forward?

**Comment by kcm** [comment]
Rebased and reopened after editing per @cbuescher's last comments.

**Comment by tvernum** [comment]
@kcm It looks like you might have rebased in the wrong direction. This branch seems to have a bunch of re-written commits from master copied into it.

e.g. The tip of your branch has:
```
commit 90c76f52667fc4acc29837f4d7f30ac4b1a59869
Author:     Ryan Ernst <ryan@iernst.net>
AuthorDate: Mon Oct 29 19:45:35 2018 -0700
Commit:     Ken MacInnis <ken.macinnis@gmail.com>
CommitDate: Tue Oct 30 15:29:19 2018 +1100

    Remove remaining line length violations in o.e.cluster (#34941)

    relates #34923, #34884
```

but the actual commit from master is:
```
commit 5dda2b0c7aa9b4fec08072afb9d64c8195deef1f
Author:     Ryan Ernst <ryan@iernst.net>
AuthorDate: Mon Oct 29 19:45:35 2018 -0700
Commit:     GitHub <noreply@github.com>
CommitDate: Mon Oct 29 19:45:35 2018 -0700

    Remove remaining line length violations in o.e.cluster (#34941)

    relates #34923, #34884
```

**Comment by kcm** [comment]
Closing in favor of #35073 as I'd prefer to get the single change in than wrestle with Git.
